pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget_test.go

```diff
@@ -109,10 +110,12 @@ var _ = Describe("Disruptionbudget", func() {
 			Expect(ok).To(BeTrue())
 			Expect(patchAction.GetName()).To(Equal("pdb-" + vmi.Name))
 			Expect(patchAction.GetPatchType()).To(Equal(types.JSONPatchType))
-
-			expectedPatch := fmt.Sprintf(`[{ "op": "replace", "path": "/spec/minAvailable", "value": 1 }, { "op": "remove", "path": "/metadata/labels/%s" }]`,
-				patch.EscapeJSONPointer(v1.MigrationNameLabel))
-			Expect(string(patchAction.GetPatch())).To(Equal(expectedPatch))
+			patches := patch.New(
+				patch.WithReplace("/spec/minAvailable", 1),
+				patch.WithRemove(fmt.Sprintf("/metadata/labels/%s", patch.EscapeJSONPointer(
+					virtv1.MigrationNameLabel))),
+			)
+			Expect(patches.GeneratePayload()).To(Equal(patchAction.GetPatch()))
```

Comment:
nit: to me the comparison elements are inverted. Usually the `Expect` contains the retrieved value, while the `Equal` the expected one.

Reply:
It is done in this way to check also the error, please see Ed's comment https://github.com/kubevirt/kubevirt/pull/11599#discussion_r1540823391